### PR TITLE
Add rolldown CLI tools

### PR DESCRIPTION
**Pull request by AlexChetverov**

### CODE REVIEW

File: test/parallel/rolldown.test.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    await setupApi();
+    await sudo.addMGATokens(sudo);
+  });
+
+  beforeEach(async () => {
+    testEthUser = new EthUser(keyring);
+  });
+
+  test("Deposit token using rolldown", async () => {
```
Comment (goncer) [inline test/parallel/rolldown.test.ts:47]: skip this test, because it will fail if some other deposits happen in the same env.

**Review by goncer — CHANGES_REQUESTED** [review]

File: test/parallel/rolldown.test.ts (hunk 1 above)
Comment (AlexChetverov) [reply test/parallel/rolldown.test.ts:47]: Done

File: test/parallel/rolldown.test.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
+/*
+ *
+ * @group rolldown
```
Comment (goncer) [inline test/parallel/rolldown.test.ts:3]: remove the group plz

Comment (goncer) [reply test/parallel/rolldown.test.ts:3]: and move this test to the rollup folder!

**Review by goncer — CHANGES_REQUESTED** [review]

File: test/parallel/rolldown.test.ts (hunk 2 above)
Comment (AlexChetverov) [reply test/parallel/rolldown.test.ts]: Done

**Review by goncer — APPROVED** [review]

**Pull request merged**